arcgis-ios-sdk-samples/Utility Network/Trace utility network/README.md

```diff
@@ -0,0 +1,57 @@
+# Trace utility network
+
+Discover connected features in a utility network using connected, subnetwork, upstream, and downstream traces.
+
+![Image of trace utility network](trace-utility-network.png)
+
+## Use case
+
+You can use a trace to visualize and validate the network topology of a utility network for quality assurance. Subnetwork traces are used for validating whether subnetworks, such as circuits or zones, are defined or edited appropriately.
+
+## How to use the sample
+
+Tap on one or more features while "Start" or "Barrier" is selected. When a junction feature is identified, you may be prompted to select a terminal. When an edge feature is identified, the distance from the tapped location to the beginning of the edge feature will be computed. Tap "Type" to select the type of trace using the action sheet. Tap "Trace" to initiate a trace on the network. Tap "Reset" to clear the trace parameters and start over.
+
+## How it works
+
+1. Create an `AGSMapView` and listen for `didTap` events on the `AGSGeoViewTouchDelegate`.
+2. Create an `AGSMap` that contains `AGSFeatureLayer`(s) that are part of a utility network.
+3. Create and load an `AGSUtilityNetwork` with the same feature service URL and map.
+4. Add an `AGSGraphicsOverlay` with symbology that distinguishes starting locations from barriers.
+5. Identify tapped features on the map and add an `AGSGraphic` that represents its purpose (starting point or barrier) at the tapped location.
+6. Create an `AGSUtilityElement` for the identified feature.
+7. Determine the type of the identified feature using `AGSUtilityNetworkSource.sourceType` passing its table name.
```

Comment:
"passing its table name" doesn't seem to make sense since `AGSUtilityNetworkSource.sourceType` is a property.

```suggestion
7. Determine the type of the identified feature using `AGSUtilityNetworkSource.sourceType`.
```